pkg/controller/nodemaintenance/nodemaintenance_controller_test.go

```diff
@@ -106,6 +106,25 @@ var _ = Describe("updateCondition", func() {
 			Expect(maintanance.Status.EvictionPods).To(Equal(2))
 			Expect(maintanance.Status.TotalPods).To(Equal(2))
 		})
+		It("owner ref should be set properly", func() {
+			r.initMaintenanceStatus(nm)
+			maintanance := &nodemaintenanceapi.NodeMaintenance{}
+			err := cl.Get(context.TODO(), req.NamespacedName, maintanance)
+			node, err := cs.CoreV1().Nodes().Get("node01", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			hasSetRef := setOwnerRefToNode(maintanance, node)
+			Expect(hasSetRef).To(Equal(true))
```

Comment:
As mentioned before: the actual code does not use the return value so I think it should be removed.
In this test it is enough to check that the owner ref is set correctly, as it is done in the following lines.

general hint: `To(Equal(true))` can be `To(BeTrue())`

Reply:
return value of stOwnerRefToNode has been removed.

Reply:
done, return value removed.